style.css

```diff
@@ -122,6 +102,81 @@ input, select {
     border: #ffffff solid 2px;
     border-radius: 3px;
 }
+.card-parent {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+.card-parent div {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.card-parent h3 {
+    display: inline-block;
+}
+.card-outer {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 30px;
+}
+.card-inner {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+}
+.card-inner div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0;
+}
+.card-inner div p {
+    margin: 0 auto;
+}
+a .anchor {
```

Comment:
this class should be called something different and it should be applied on the div instead and there should be no space because `a .anchor` is different than `a.anchor` (as discussed)